templates/partners/kubernetes.html

```diff
@@ -0,0 +1,86 @@
+        <h3 class="p-card__title" style="padding-top: 3rem;">In partnership with Google</h3>
+        <p class="p-card__content u-sv3">Google and Canonical together enable smooth hybrid operations between Google’s Container Engine (GKE) and Elastic Container Service for Kubernetes (EKS) services with Ubuntu worker nodes, and the Charmed Distribution of Kubernetes. Choose Charmed Kubernetes to be sure your container workloads can migrate to GKE or EKS thanks to our kernel-to-k8s alignment.</p>
+        <!-- Add link url -->
+        <a href="" class="p-button--positive">Try GKE with Ubuntu now</a>
+        <p>
+          <a href="" class="p-link--external">Learn about EKS with Ubuntu</a>
+        </p>
+      </div>
+      <div class="p-card--no-border col-4">
+        <img class="p-card__thumbnail-large" src="https://assets.ubuntu.com/v1/45012d4e-jfrog.png" alt="jfrog image">
+        <h3 class="p-card__title" style="padding-top: 3rem;">JFrog Artifactory</h3>
+        <p class="p-card__content u-sv3">JFrog Artifactory is the enterprise-ready Kubernetes registry fully integrated with both Rancher and the Charmed Distribution of Kubernetes. Long known to developers, Artifactory provides the governance, audit capability and artifact management. Canonical partners with JFrog to integrate CDK with a single point and click installation using conjure-up.</p>
+        <!-- Add link url -->
+        <a href="" class="p-button--positive">Watch the JFrog Artifactory webinar</a>
+      </div>
+    </div>
+  </section>
+  <section class="p-strip--light is-slanted--top-right is-deep">
+    <div class="row">
+      <p class="p-heading--two">Partner benefits:</p>
+      <ul class="p-list is-split">
+        <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt </li>
+        <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt </li>
+        <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt </li>
+        <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt </li>
+        <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt </li>
+        <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt </li>
+      </ul>
+    </div>
+  </section>
+  <section class="p-strip--mid-light is-deep">
+    <div class="row p-divider">
+      <div class="col-6 p-divider__block">
+        <div class="p-media-object">
+          <img src="https://assets.ubuntu.com/v1/2cdff5cc-magnifier-find-a-partner-footer.svg" class="p-media-object__image">
+          <div class="p-media-object__details">
+            <h1 class="p-media-object__title">Find a partner</h1>
```

Comment:
This should also be a `<h2>`, semantically it doesn't belong on the same level as the main title in the page heirachy.

It looks smaller than in the design, too. Maybe `.p-heading--two` would work?

![Screenshot 2019-10-07 at 17 08 05](https://user-images.githubusercontent.com/2376968/66328707-0f398380-e925-11e9-80d6-3d41e4eda4a7.png)